### PR TITLE
ci: queue rules update to handle prs 

### PR DESCRIPTION
**Pull request by Rakshith-R**
This commit adds queue rules to prevent closed,draft
prs with DNM label from entering the queue.

Signed-off-by: Rakshith R <rar@redhat.com>

### CODE REVIEW

**Review by humblec — DISMISSED** [review]

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
DCO is failing and also mention in the PR description/commit message why this change is required.

**Review by humblec — APPROVED** [review]

**Comment by humblec** [comment]
@Madhu-1 ptal.. thanks!

File: .mergify.yml
Comment (Madhu-1) [inline .mergify.yml:22]: add a condition to have approval from the maintainers and contributors to make it more clear?

Comment (humblec) [reply .mergify.yml:22]: https://github.com/ceph/ceph-csi/pull/2763 address or club the changes.. ptal.

**Comment by nixpanic** [comment]
Is #2764 a replacement for this?

**Pull request closed without merging**
